tests/integration/pilot/analysis/analysis_test.go

```diff
@@ -32,11 +32,15 @@ import (
 )
 
 func TestAnalysisWritesStatus(t *testing.T) {
-	framework.NewTest(t).Run(func(ctx framework.TestContext) {
-		ns := setupModifiedBookinfo(t, ctx)
-		retry.UntilSuccessOrFail(t, func() error { return doTest(t, ctx, ns) },
-			retry.Delay(250*time.Millisecond), retry.Timeout(30*time.Second))
-	})
+	framework.NewTest(t).
+		LabelFeatures("Usability.Observability.Status").
```

Comment:
Is there any concern with type-os? If there are tons of features that extend from `Usability`, I expect it's just a matter of time until someone misspells it `Useability`, for example.

Reply:
Yes.  These should be heirarchical constants, but I didn't want to write the code generator for that until we have high level agreement on direction (see line 37).